code/_onclick/hud/pai.dm

```diff
@@ -123,12 +213,48 @@
 	using.screen_loc = ui_borg_language_menu
 	static_inventory += using
 
-//Take image
+// Host Monitor
+	using = new /obj/screen/pai/host_monitor()
+	using.screen_loc = ui_pai_host_monitor
+	static_inventory += using
+
+// Crew Manifest
+	using = new /obj/screen/pai/crew_manifest()
+	using.screen_loc = ui_pai_crew_manifest
+	static_inventory += using
+
+// Laws
+	using = new /obj/screen/pai/state_laws()
+	using.screen_loc = ui_pai_state_laws
+	static_inventory += using
+
+// PDA message
+	using = new /obj/screen/pai/pda_msg_send()
+	using.screen_loc = ui_pai_pda_send
+	static_inventory += using
+
+// PDA log
+	using = new /obj/screen/pai/pda_msg_show()
+	using.screen_loc = ui_pai_pda_log
+	static_inventory += using
+
+// Take image
 	using = new /obj/screen/pai/image_take()
 	using.screen_loc = ui_pai_take_picture
 	static_inventory += using
 
-//View images
+// View images
 	using = new /obj/screen/pai/image_view()
 	using.screen_loc = ui_pai_view_images
-	static_inventory += using
\ No newline at end of file
+	static_inventory += using
+
+// Radio
+	using = new /obj/screen/pai/radio()
+	using.screen_loc = ui_borg_radio
+	static_inventory += using
+
+	for(var/obj/screen/pai/button in static_inventory)
+		if(button.required_software)
+			button.color = owner.software.Find(button.required_software) || "#808080"
```

Comment:
I thought about this for a sec and even though I wrote this, this is more correct because it doesn't involve setting color to a number.. sorry!

```suggestion
			button.color = owner.software.Find(button.required_software) ? null : "#808080"
```